packages/react-dom/src/client/ReactDOMRoot.js

```diff
@@ -30,6 +29,8 @@ export type RootOptions = {
   ...
 };
 
+import {findHostInstanceWithNoPortals} from 'react-reconciler/src/ReactFiberReconciler';
+import {registerMutableSourceForHydration} from 'react-reconciler/src/ReactMutableSource';
```

Comment:
Could you export this via the reconciler module? Then you don't to configure an additional forked entry point

Reply:
Sure.